src/Stratis.Bitcoin.IntegrationTests/Wallet/WalletTests.cs

```diff
@@ -205,12 +206,10 @@ public void Given_TheNodeHadAReorg_And_WalletTipIsBehindConsensusTip_When_ANewBl
         {
             using (NodeBuilder builder = NodeBuilder.Create(this))
             {
-                CoreNode stratisSender = builder.CreateStratisPowNode(this.network).WithWallet().Start();
```

Comment:
Does WithReadyBlockchainData replace WithWallet?

Reply:
Yes, the wallet is included in the ReadyData 👍 So no need to create it in the test.

Reply:
@dangershony there is a wallet file inside the data zips, so I believe it does.